src/c++/library/http_client.cc

```diff
@@ -939,6 +941,16 @@ InferResultHttp::RawData(
   return Error::Success;
 }
 
+Error
+InferResultHttp::IsFinalResponse(bool* is_final_response) const
```

Comment:
For HTTP we currently don't have a final response parameter.

Reply:
This always returns true. It needs to be implemented here to follow the pattern in `InferResult` of defining all public member methods as pure virtual.